src/V1Meta.js

```diff
@@ -54,4 +54,10 @@ export default class V1Meta {
         const headers = createHeaderObj(this.authHeader);
         return Promise.resolve(this.postFn(url, null, headers));
     }
+
+    queryDefinition(assetType) {
+        const url = `${this.urls.meta()}/${assetType}`;
+        const headers = createHeaderObj(this.authHeader);
+        return Promise.resolve(this.postFn(url, null, headers));
```

Comment:
Also, we might as well add the accept header to be JSON in what is returned to the postFn. All other functions return JSON in the SDK and there isn't a way to override a specific one to be something different.